packages/main/src/themes/base/Card-parameters.css

```diff
@@ -6,6 +6,7 @@
 	--_ui5_card_content_padding: 1rem;
 	--_ui5_card_header_hover_bg: var(--sapList_Hover_Background);
 	--_ui5_card_header_active_bg: var(--_ui5_card_header_hover_bg);
+	--_ui5_card_header_border: 1px solid var(--_ui5_card_header_border_color);
```

Comment:
Change all added lines from px to rem.

Reply:
it is OK to be in px when is 1px. For different values rem is used